Fixes #23463: Add an upgrade node about the removal of "move" delete mode (#1004)
Normation/rudder-doc: src/reference/modules/installation/pages/upgrade/notes.adoc

after upgrade, as they will stop
working in a future Rudder version.

=== Node delete mode is "erase"

> Historically, Rudder has supported an node delete mode where node information

Reviewer: ```suggestion
Historically, Rudder has supported a node delete mode where node information
```

> were move in a dedicated LDAP branch and not deleted in addition to the 

Reviewer: ```suggestion
was moved in a dedicated LDAP branch and not deleted in addition to the 
```

real deletion of information. The choice was configured by property
`rudder.nodes.delete.defaultMode` and the default was `erase` since Rudder 7.2.
At the same time, the `move` mode was deprecated.

> As of Rudder 8.0, the old `move` mode is removed and node information are erased

Reviewer: ```suggestion
As of Rudder 8.0, the old `move` mode is removed and node information is erased
```

> from base when a node is deleted.

Reviewer: ```suggestion
from the database when a node is deleted.
```
